dll/win32/shimgvw/shimgvw.c

```diff
@@ -609,14 +611,31 @@ ImageView_DrawImage(HWND hwnd)
 
     if (GetClientRect(hwnd, &rect))
     {
-        FillRect(hdc, &rect, (HBRUSH)GetStockObject(WHITE_BRUSH));
-
         ZoomedWidth = (ImageWidth * ZoomPercents) / 100;
         ZoomedHeight = (ImageHeight * ZoomPercents) / 100;
 
         x = (rect.right - ZoomedWidth) / 2;
         y = (rect.bottom - ZoomedHeight) / 2;
 
+        white = GetStockObject(WHITE_BRUSH);
```

Comment:
Why white? Shouldn't it be of the default (background) color of windows?

Reply:
- It looks good with this color :)
- It was done with this color before
- Also this color is used by shimgvw in Windows

Reply:
Thanks for the precisions :)